### PR TITLE
Master

### PR DESCRIPTION
**Pull request by darkweb907**
Added spline design tools for interactive 3D experiences or animations

### CODE REVIEW

File: database/frontend/themes-templates.json [hunk 1]
```diff
@@ -19,5 +19,19 @@
...
+    {
+        "name": "landing page template",
+        "description": "Simple, free, fully responsive one-page sites for pretty much anything.",
+        "url": "https://carrd.co",
+        "category": "frontend",
+        "subcategory": "themes-templates"
+    },
+    {
+        "name": "Creative-tim",
+        "description": "fully responsive UI Kits, Templates and Dashboards built on top of Bootstrap, React",
```
Comment (CBID2) [inline database/frontend/themes-templates.json:32]:
```suggestion
        "description": "Fully responsive UI kits, templates, and dashboards that are built on top of Bootstrap and React",
```

File: database/youtube/android.json [hunk 2]
```diff
@@ -5,4 +5,12 @@
     "category": "youtube",
     "subcategory": "android",
     "language": "english"
+},
+{
+    "name": "Android Developer",
+    "description": "This channel provides free programming languages for Android  development and more.",
```
Comment (CBID2) [inline database/youtube/android.json:11]:
```suggestion
    "description": "This channel provides free programming courses on Android development and more.",
```

File: database/youtube/web-development.json [hunk 3]
```diff
@@ -61,5 +61,22 @@
         "category": "youtube",
         "subcategory": "web-development",
         "language": "hindi"
+    },
+    {
+        "name": "DesignCourse",
+        "description": "This channel will provide and take you from zero to a fullstack developer.you will learn UI/UX, Frontend Dev",
```
Comment (CBID2) [inline database/youtube/web-development.json:67]:
```suggestion
        "description": "This channel will take you from zero to a fullstack developer. You will also learn UI/UX and Frontend Dev languages",
```

File: database/youtube/web-development.json [hunk 4]
```diff
@@ -61,5 +61,22 @@
...
+        "url": "https://www.youtube.com/@DesignCourse/featured",
+        "category": "youtube",
+        "subcategory": "web-development",
+        "language": "english"
+    },
+    {
+        "name": "Bro Code",
+        "description": "This channel is dedicated on providing free education on coding tutorial for both beginners and pro",
```
Comment (CBID2) [inline database/youtube/web-development.json:75]:
```suggestion
        "description": "This channel is dedicated to providing free coding tutorials for both beginners and pros",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @darkweb907! I just added some suggestions to improve your PR! 😄 I hope you find them helpful! 👍🏾

**Review by CBID2 — APPROVED** [review]
Looks great! :) LGTM

**Comment by rupali-codes** [comment]
Thank you chrissy. 
@darkweb907 could you please resolve conflicts

**Pull request merged**